docs/guide.md

```diff
@@ -142,6 +142,16 @@ API keys can be created, viewed and revoked programmatically by manipulating the
 !!! danger
     To prevent leaking API keys, you must only give the `key` **to the client that triggered its generation**. In particular, **do not keep any trace of it on the server**.
 
+- To retrieve an `APIKey` instance based on its generated key (which is not stored in the database) use the `.get_from_key()` method on the `APIKey` objects manager instead of `.get()`:
+
+```python
+>>> from rest_framework_api_key.models import APIKey
+>>> # Generated keys follow this format
+>>> generated_key = "zx6UGvip.XakvjxQJUSRCHIEdMUDtwZtEnb4YLHaL"
```

Comment:
Let's use a redacted key? :-)

```suggestion
>>> generated_key = "apr3fixb.longstringofcharactersandl3tt3rs"
```

Reply:
Smart thinking 😅Thankfully at least this key shouldn't be valid for anything, but probably best not to give anyone ideas. I swapped out this code snippet for the one you provided above, so this issue should be moot now.